evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -28,213 +26,6 @@ def compare_two_tables(table_1, table_2):
         assert row.strip().split() == row_answer.strip().split()
 
 
-test_features = [[1], np.ones((15, 1)), pd.DataFrame({"a": [1, 2, 3], "b": [1, 2, 3]}).iloc[0],
```

Comment:
We have coverage in `test_explain_predictions_best_worst_and_explain_predictions`